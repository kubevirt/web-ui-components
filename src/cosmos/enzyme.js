import createContext from 'react-cosmos-test/enzyme';

import paths from '../../config/paths';

export default function createTestContext(args) {
  args.proxies = args.proxies || []; // workaround of react-cosmos-test bug
  return createContext({
    ...args,
    cosmosConfigPath: `${paths.config}/cosmos.config.js`,
  });
}
