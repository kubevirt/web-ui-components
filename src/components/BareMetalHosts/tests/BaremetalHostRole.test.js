import React from 'react';
import { shallow } from 'enzyme';

import { BaremetalHostRole } from '..';

import BaremetalHostRoleFixture from '../fixtures/BaremetalHostRole.fixture';

const testBaremetalHostRole = () => <BaremetalHostRole {...BaremetalHostRoleFixture[0].props} />;

describe('<BaremetalHostRole />', () => {
  it('renders a role for host', () => {
    const component = shallow(testBaremetalHostRole());
    expect(component).toMatchSnapshot();
  });
});
