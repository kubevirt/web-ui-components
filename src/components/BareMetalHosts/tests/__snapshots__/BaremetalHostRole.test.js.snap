// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaremetalHostRole /> renders a role for host 1`] = `"worker"`;
